k8s/crds.yaml

```diff
@@ -1361,6 +1361,8 @@ spec:
                 type: boolean
               salvageRequested:
                 type: boolean
+              unmapMarkSnapChainRemovedEnabled:
```

Comment:
Why there is no `UnmapMarkSnapChainRemovedEnabled` in the engine status?

Reply:
Sorry. I just manually updated the CRD in repo `longhorn` and forgot to update this.

Reply:
I see, but next time you can use https://github.com/longhorn/longhorn-manager/blob/master/k8s/generate_code.sh#L70 to update the crds.yaml directly.